sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/SchedulerApiServerTest.java

```diff
@@ -28,7 +28,7 @@ public void testApiServerReady() throws Exception {
         SchedulerApiServer schedulerApiServer = new SchedulerApiServer(12345, Collections.emptyList(),
                 Duration.ofMillis(TIMEOUT_MILLIS));
         new Thread(schedulerApiServer).start();
-        Thread.sleep(TIMEOUT_MILLIS * 10);
```

Comment:
Github lost my comments on this. Thanks github.

Yes, make the timeout in the constructor larger (30 seconds, the auto-kill of our unit tests), then just poll on it.